lib/sandboxes.ts

```diff
@@ -321,8 +306,8 @@ function handleSandboxCreateError({
       category: 'VALIDATION_ERROR',
       subCategory: 'SandboxErrors.NUM_STANDARD_SANDBOXES_LIMIT_EXCEEDED_ERROR',
     }) &&
-    err.error &&
-    err.error.message
+    'error' in err &&
+    err.error instanceof Error
```

Comment:
I think this if statement might be unreachable code.  [if the `isSpecifiedError` is true](https://github.com/HubSpot/hubspot-local-dev-lib/blob/2b22bad59719f3d3df34da0a52f1656eb2b7c5dd/errors/index.ts#L5-L20C11), it is a [`HubSpotHttpError` which doesn't have a `error` member on it](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/models/HubSpotHttpError.ts#L8)

Reply:
Gotcha, I wasn't sure if we always knew the shape of `HubSpotHttpError`s